ami-build/agent-setup-minimal.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -eu
+
+set -o pipefail
+
+AGENT_VERSION=3.218.0
```

Comment:
Is this a stable 3.x? Does it mean I can remove the workaround from Salvo and we can upgrade main pools to Ubuntu Jammy?

I did not notice any openssl lib hacks in the scripts, so I am hopeful ;)

Reply:
Nice, I will use the same for Salvo. Thank you.